### PR TITLE
RSDK-3128 - Remote control (Base) example

### PR DESCRIPTION
**Pull request by clintpurser**
- adds logo for example app
- adds base (rc) screen
- widgetizes stream widget and base joystick widget.

I don't love the way i'm passing in a camera and stream and base into the widgets / new screens. I think passing around a RescourceName would be cleaner, that will require a little bit of refactoring which I want to do in a separate follow up PR, as to not spend too much time on this right now.

### CODE REVIEW

File: example/viam_example_app/lib/main.dart [hunk 1]
```diff
@@ -74,6 +75,7 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   bool _loggedIn = false;
   bool _loading = false;
+  late ResourceName? _cameraName;
```
Comment (njooma) [inline example/viam_example_app/lib/main.dart:78]: This should either be `late` or `optional`, no need for it to be both

**Review by njooma — APPROVED** [review]

File: example/viam_example_app/lib/main.dart (hunk 1 above)
Comment (clintpurser) [reply example/viam_example_app/lib/main.dart:78]: good catch, thank you!

File: .DS_Store
Comment (cheukt) [inline .DS_Store:1]: delete?

**Review by cheukt — APPROVED** [review]

File: .DS_Store
Comment (clintpurser) [reply .DS_Store:1]: 👀 huh - yeah lets axe this

Comment (njooma) [reply .DS_Store:1]: Added it here: https://github.com/viamrobotics/viam-flutter-sdk/pull/45/commits/7f462a98744bad564f9daf8560a7c03c1a01f4ad

File: example/viam_example_app/.metadata [hunk 2]
```diff
@@ -13,26 +13,11 @@ project_type: app
...
-      create_revision: 90c64ed42ba53a52d18f0cb3b17666c8662ed2a0
-      base_revision: 90c64ed42ba53a52d18f0cb3b17666c8662ed2a0
-    - platform: web
-      create_revision: 90c64ed42ba53a52d18f0cb3b17666c8662ed2a0
-      base_revision: 90c64ed42ba53a52d18f0cb3b17666c8662ed2a0
-    - platform: windows
-      create_revision: 90c64ed42ba53a52d18f0cb3b17666c8662ed2a0
-      base_revision: 90c64ed42ba53a52d18f0cb3b17666c8662ed2a0
+      create_revision: 84a1e904f44f9b0e9c4510138010edcc653163f8
+      base_revision: 84a1e904f44f9b0e9c4510138010edcc653163f8
```
Comment (clintpurser) [inline example/viam_example_app/.metadata:20]: i think we can remove this file, i'll do so in a follow up. probably lots to clean up in the example app.

**Pull request merged**